packages/infra/lib/lambdas-nested-stack.ts

```diff
@@ -135,11 +135,13 @@ export class LambdasNestedStack extends NestedStack {
       bundleBucket: props.medicalDocumentsBucket,
       conversionsBucket: this.fhirConverterConnector.bucket,
       envType: props.config.environmentType,
+      secrets: props.secrets,
       sentryDsn: props.config.lambdasSentryDSN,
       alarmAction: props.alarmAction,
       appId: props.appConfigEnvVars.appId,
       configId: props.appConfigEnvVars.configId,
       bedrock: props.config.bedrock,
+      posthogSecretKeyName: props.config.analyticsSecretNames?.POST_HOG_API_KEY_SECRET,
```

Comment:
OOC, why are we passing this as a parameter here but on `fhir-converter-connector.ts` we have it directly on the function that uses it?